test/ArborX_EnableDeviceTypes.hpp.in

```diff
@@ -1,10 +1,21 @@
+/****************************************************************************
+ * Copyright (c) 2017-2021 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
 #ifndef ARBORX_ENABLE_DEVICE_TYPES_HPP
 #define ARBORX_ENABLE_DEVICE_TYPES_HPP
 
 #include <Kokkos_Core.hpp>
 
 #include <tuple>
 
-using ARBORX_DEVICE_TYPES = std::tuple<@ARBORX_DEVICE_TYPES@>;
+using ARBORX_DEVICE_TYPES = std::tuple<@ARBORX_DEVICE_TYPES @>;
```

Comment:
Why did you add a space here

Reply:
For some reason, my vim clang-format plugin does that sometimes. Fixed.